data_structures/src/transaction.rs

```diff
@@ -259,6 +279,29 @@ impl DRTransaction {
             .map(|tx_idx| TxInclusionProof::new(tx_idx, txs))
     }
 
+    /// Returns the weight that a data request transaction.
+    /// This is the weight that will be used to calculate
```

Comment:
A link to the WIP might be of help, otherwise the equation seems that it comes out of nowhere

Reply:
Link to the PR? Because there are not a wip right now

Reply:
True. :sweat_smile: